### PR TITLE
Add action to Murmur

### PR DESCRIPTION
**Pull request by n1mda**
Allow user to tap the Whistle to run a closure

### CODE REVIEW

File: Source/WhistleFactory.swift [hunk 1]
```diff
@@ -153,4 +164,11 @@ public class WhistleFactory: UIViewController {
       hide()
     }
   }
+    
+  // MARK: - Gesture methods
+    
+  @objc private func handleTapGestureRecognizer() {
```
Comment (RamonGilabert) [inline Source/WhistleFactory.swift:170]: Curious about the `@objc` selector, do you need it? :)

Comment (n1mda) [reply Source/WhistleFactory.swift:170]: Yes :) Explained here: http://stackoverflow.com/a/24007718/1093865
Basically without `@objc` it throws an error: `Argument of '#selector' refers to a method that is not exposed to Objective-C`

**Comment by RamonGilabert** [comment]
Thanks for the PR @n1mda :)

**Comment by n1mda** [comment]
You're welcome :) Thank you for fast QA 👍

**Pull request merged**